### ISSUE TITLE
Cannot read property 'format' of undefined. Error when including SVG format?

### ISSUE THREAD

**Issue by granttransition** [11ty/eleventy-img#61]
```"@11ty/eleventy-img": "^0.7.8"```
```
const Image = require("@11ty/eleventy-img");

async function imageShortcode(src, alt, sizes) {
  let metadata = await Image(src, {
    widths: [300, 600],
    formats: ["avif", "webp", "jpeg", "svg"],
    svgShortCircuit: true,
    outputDir: "./src/_includes/assets/img/",
    urlPath: "./assets/img/"
  });

  let imageAttributes = {
    alt,
    sizes,
    loading: "lazy",
    decoding: "async",
  };

  // You bet we throw an error on missing alt in `imageAttributes` (alt="" works okay)
  return Image.generateHTML(metadata, imageAttributes);
}

module.exports = eleventyConfig => {
eleventyConfig.addNunjucksAsyncShortcode("image", imageShortcode);
}
```

HTML

```
      {% image "src/_assets/img/social-share-img.jpg", "photo of my cat", "(min-width: 30em) 50vw, 100vw" %}
      {% image "src/_assets/svg/arkell.svg", "photo of my cat", "(min-width: 30em) 50vw, 100vw" %}
```

Hi,

When I include `svg` in my formats an error is thrown saying `Cannot read property 'format' of undefined`. If I remove `svg` everything runs as expected.

This is most likely user error but I can't figure out what I'm not doing correctly?

**Comment by zachleat** [comment]
Maybe a duplicate of #48?

**Comment by zachleat** [comment]
Also marginally related #72.

These are both fixed in upcoming 0.8.0—please retest and report back if you’re still seeing issues after that release!

**Comment by zachleat** [comment]
> This is an automated message to let you know that a helpful response was posted to your issue and for the health of the repository issue tracker the issue will be closed. This is to help alleviate issues hanging open waiting for a response from the original poster.

> If the response works to solve your problem—great! But if you’re still having problems, do not let the issue’s closing deter you if you have additional questions! Post another comment and I will reopen the issue. Thanks!
